README.md

```diff
@@ -15,7 +15,7 @@ and quantities through the [Python Pint library](https://pypi.org/project/Pint/)
 
 * Python 3.8+
 * Setuptools for installing dependencies
-* Other Python libraries (installed automatically when using pip/easy_install): requests, pytz, pint, arrow
+* Other Python libraries (installed automatically when using pip/easy_install): requests, pytz, pint, arrow, pydantic
```

Comment:
```suggestion
* Other Python libraries (installed automatically when using pip): requests, pytz, pint, arrow, pydantic
```

Reply:
good to get rid of easy install as its [deprecated](https://packaging.python.org/en/latest/discussions/pip-vs-easy-install/)!